### DISCUSSION TITLE
Add support for Web Share API Level 2

### DISCUSSION THREAD

**Discussion by jonathon-m** [ionic-team/capacitor#1704] [Ideas]
In Chrome 75 on Android, it is now possible to share files using the native share sheet. We can only assume that this will be available in other browser soon, since it builds on the current Web Share API standard. 

An example of how this is done can be found [here](https://developers.google.com/web/updates/2019/05/web-share-files).

This is particularly useful for anyone building PWAs.

**Comment by mehmetoguzderin** [comment]
Is it possible to have this be compatible with Filesystem API so we can just pass their URI and Ionic handles the rest?

**Comment by mattsteve** [comment]
It's a shame, I would like to use the capacitor plugins but it's difficult when they miss basic functionality like this. Guess I have to fall back to https://www.npmjs.com/package/cordova-plugin-x-socialsharing.
